src/cartservice/Services/CartServiceController.cs

```diff
@@ -45,11 +45,6 @@ public async override Task<Empty> EmptyCart(EmptyCartRequest request, ServerCall
         {
             await _cartStore.EmptyCartAsync(request.UserId);
             return Empty;
-        } 
-        public async void Start()
-        {
-            Console.WriteLine("Starting cart service...");
-            await _cartStore.InitializeAsync();

```

Comment:
Where do you initialize cartStore?

Reply:
This method was not used, the initialization is done in `Startup.cs`: https://github.com/GoogleCloudPlatform/cloud-ops-sandbox/blob/mathieu-benoit/update-cartservice/src/cartservice/Startup.cs#L52